Jg feat implements self contained python template

jjgouveia opened the pull request:
Este pull request inclui várias atualizações no serviço WoolBall Speech-to-Text em diferentes templates de linguagem (C#, Node.js e Python). As mudanças se concentram na atualização dos endpoints da API, nas chaves de dados do formulário usadas nas requisições, além da adição de nova documentação e exemplos de uso para o template em Python.

## Atualizações de Endpoints da API:
- Atualizado o endpoint para buscar modelos disponíveis de `/speech-to-text-models` para `/models/speech-to-text` nos arquivos C# e Node.js. [[1]](diffhunk://#diff-61777ba794d894d73f868d376af45b29c2c59d5dd698c559f0d71ffa89bebc6fL149-R149) [[2]](diffhunk://#diff-cef9ca3cf957590811f616ad7d2dca08961940fc7d04672595d5d1fbb8f04084L68-R70)
- Alterada a chave de dados do formulário de `file` para `audio` no upload de arquivos de áudio nos arquivos C#, Node.js e Next. [[1]](diffhunk://#diff-61777ba794d894d73f868d376af45b29c2c59d5dd698c559f0d71ffa89bebc6fL101-R101) [[2]](diffhunk://#diff-cef9ca3cf957590811f616ad7d2dca08961940fc7d04672595d5d1fbb8f04084L25-R25) [[3]](diffhunk://#diff-6d984012ceb512f58d5c6e525e9cc9906fb046b6599cff43d35fd27f4d2a2334L34-R34)

## Documentação e Exemplos:
- Adicionado um README abrangente para o template Python, incluindo instruções de instalação, exemplos de uso e formatos suportados.
- Criado um script detalhado `usage.py` com exemplos de várias funcionalidades do serviço WoolBall Speech-to-Text, como transcrição a partir de URLs e arquivos, obtenção de modelos disponíveis e manipulação de diferentes tipos de entrada.

jjgouveia committed:
feat: provê o template de implementação para python (self contained)

jjgouveia committed:
refact: corrige uma pequena inconsistencia no service a adiciona um arquivo mp3 para testes

jjgouveia committed:
fix: ajusta um campo do form do método `transcribeFromFile` e corrige a url de listagem dos models no service do node

jjgouveia committed:
fix: corrige os métodos que recebem um arquivo local para transcrição e que estavam apontando para uma variável incorreta. Corrige também a rota que lista os modelos disponíveis

jjgouveia committed:
fix: remove o token da api_key

Joao620 commented:
A pedido do @eduardoworrel , fiz uma review do pull request. Só tenho dois pontos:
1. a biblioteca `asyncio` tá sendo importada duas vezes no arquivo de `usage.py`
2. Poderia ser colocados em inglês os logs na implementação em python? Para seguir que já está tudo em inglês no código

jjgouveia committed:
Localizes error messages and removes redundant import

Translates error logging messages from Portuguese to English
Removes unnecessary asyncio import in usage script

Improves code readability and removes unused import

jjgouveia commented:
> A pedido do @eduardoworrel , fiz uma review do pull request. Só tenho dois pontos:
> 
> 1. a biblioteca `asyncio` tá sendo importada duas vezes no arquivo de `usage.py`
> 2. Poderia ser colocados em inglês os logs na implementação em python? Para seguir que já está tudo em inglês no código

Feito!

eduardoworrel commented:
ótimo trabalho!

Estamos reestruturando o input e output da API, por isso por enquanto os PRs estão congelados
